doc/Gridded_Datasets_I.ipynb

```diff
@@ -318,11 +318,8 @@
    "source": [
     "%%opts Curve [aspect=2 xticks=4 xrotation=15] Points (color='k')\n",
     "temp_curve = hv.Curve(xr_dataset.select(longitude=0, latitude=10), kdims=['time'])\n",
-    "\n",
-    "temp_maps = [cb.to(gv.Image,['longitude', 'latitude']) * gv.Points([(0,10)], crs=crs.PlateCarree()) \n",
-    "             for cb in [xr_dataset, air_temperature]]\n",
-    "\n",
-    "hv.Layout(temp_maps + [temp_curve]).cols(2)"
+    "temp_map = xr_dataset.to(gv.Image,['longitude', 'latitude']) * gv.Points([(0,10)], crs=crs.PlateCarree())\n",
+    "temp_map + temp_curve"
    ]
```

Comment:
The new version looks much better, though  I can't see why the convoluted one in the old version was ever needed!